mkdocs.yml

```
@@ -7,13 +7,29 @@ repo_url: https://github.com/aquasecurity/trivy
 edit_uri: ""
 
 nav:
-  - HOME: index.md
-  - Getting started:
-      - Overview: getting-started/overview.md
-      - Installation: getting-started/installation.md
-      - Quick Start: getting-started/quickstart.md
-      - Further Reading: getting-started/further.md
-  - Docs:
+  - Getting Started: 
+      - Overview: index.md
+      - Quick Start: home/quickstart.md
+      - Installation: home/installation.md
+  - Tutorials:
+      - Overview: tutorials/overview.md
+      - Trivy Kubernetes: 
```

nit: IMO, we don't need `Trivy` everywhere like `Trivy AWS` and `Trivy CircleCI` 😆  

```suggestion
      - Kubernetes: 
```

---

Not fixed yet?